modules/etcd-increase-db.adoc

```diff
@@ -0,0 +1,180 @@
+etcd-ci-ln-qkgs94t-72292-9clnd-master-0                  3/4     Running             0          20s
+etcd-ci-ln-qkgs94t-72292-9clnd-master-0                  4/4     Running             0          20s
+...
+----
+
+. Review the values by entering the following command:
++
+[source,terminal]
+----
+oc describe -n openshift-etcd pod/<ETCD_PODNAME> | grep "ETCD_QUOTA_BACKEND_BYTES"
+----
++
+The values might not have changed from the default value of `8`.
++
+.Example output
+[source,terminal]
+----
+ETCD_QUOTA_BACKEND_BYTES:                               8589934592
+ETCD_QUOTA_BACKEND_BYTES:                               8589934592
+ETCD_QUOTA_BACKEND_BYTES:                               8589934592
+ETCD_QUOTA_BACKEND_BYTES:                               8589934592
+ETCD_QUOTA_BACKEND_BYTES:                               8589934592
+----
+
+[id="etcd-change-db-size-troubleshooting_{context}"]
+== Troubleshooting
+
+If you encounter issues when you try to increase the database size for etcd, the following troubleshooting steps might help.
+
+[id="etcd-troubleshoot-small-value_{context}"]
+=== Value is too small
+
+If the value that you specify is less than 8, you will see an error message. In the following example, the value of 5 is specified:
+
+[source,terminal]
+----
+oc patch etcd/cluster --type=merge -p '{"spec": {"backendQuotaGiB": 5}}'
+----
+
+The output shows an error:
+
+.Example output
+[source,terminal]
+----
+The Etcd "cluster" is invalid: 
+* spec.backendQuotaGiB: Invalid value: 5: spec.backendQuotaGiB in body should be greater than or equal to 8
+* spec.backendQuotaGiB: Invalid value: "integer": etcd backendQuotaGiB may not be decreased
+----
+
+To resolve this issue, specify an integer between 8 and 32.
+
+[id="etcd-troubleshoot-large-value_{context}"]
+=== Value is too large
+
+If the value that you specify is greater than 32, you will see an error message. In the following example, the value of 64 is specified:
+
+[source,terminal]
+----
+oc patch etcd/cluster --type=merge -p '{"spec": {"backendQuotaGiB": 64}}'
+----
+
+The output shows an error:
+
+.Example output
+[source,terminal]
+----
+The Etcd "cluster" is invalid: spec.backendQuotaGiB: Invalid value: 64: spec.backendQuotaGiB in body should be less than or equal to 32
+----
+
+To resolve this issue, specify an integer between 8 and 32.
+
+[id="etcd-troubleshoot-decreasing-value_{context}"]
+=== Value is decreasing
+
+If the value is set to a valid value between 8 and 32 and you try to decrease the value, you will see an error message because the value can only increase. In the following example, the value is set to 10:
+
+[source,terminal]
+----
+oc patch etcd/cluster --type=merge -p '{"spec": {"backendQuotaGiB": 10}}'
+----
+
+The output shows that the change succeeded:
+
+.Example output
+[source,terminal]
+----
+etcd.operator.openshift.io/cluster patched
+----
+
+After the rollout is completed, the value is decreased to 8:
```

Comment:
Instead of setting a value as 10 and re-setting the value as 8, how about showing a current value as 10. And then setting a new value as 8.

Something like this:

--------------

1. Check to see what the current value is by entering the following command:

$ oc describe etcd/cluster | grep "Backend Quota"

Example output

Backend Quota Gi B: 10

2. Decrease the disk quota value by entering the following command:

oc patch etcd/cluster --type=merge -p '{"spec": {"backendQuotaGiB": 8}}'

.Example error message

The Etcd "cluster" is invalid: spec.backendQuotaGiB: Invalid value: "integer": etcd backendQuotaGiB may not be decreased


To resolve this issue, specify an integer greater than 10.